src/install/PackageManager/PackageManagerOptions.zig

```diff
@@ -539,6 +544,39 @@ pub fn load(
             this.node_linker = node_linker;
         }
 
+        if (cli.target_os) |os_str| {
+            if (strings.eqlComptime(os_str, "*")) {
```

Comment:
npm doesn't support '*' for all

Reply:
Yeah, but also is there anything wrong with it. Note i did add it before I realised that using one override resets the rest to allowing all.

Reply:
I see. I'm not super against adding it. It does seem useful too, because not setting the flag does not equal '*', so you would need to list each out.